spec/features/users/omniauth_request_spec.rb

```diff
@@ -164,33 +175,13 @@
     end
   end
 
-  describe "when an existing user already owns an authentication" do
```

Comment:
removing this test because it's impossible to get the UX into this state anymore. This should be perhaps a request spec but not a feature test.